components/ai_chat/resources/page/components/notices/notices.module.scss

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+.notice {
+  position: relative;
+  border-radius: var(--leo-radius-m);
+  background-color: var(--leo-color-container-highlight);
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  overflow: hidden;;
+  color: var(--leo-color-text-secondary);
+}
+
+.illustration {
+  flex: 0 0 auto;
+  width: 40%;
+  max-width: 260px;
+  background-color: #FF815C;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0 3%;
```

Comment:
is the `3%` from Figma? It feels slightly odd

Reply:
The calculation at 680px is, yes. Felt right to me to vary the padding based on width